python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -646,6 +646,20 @@ def get_asset_deps(
     Dict[str, List[str]],
     Dict[str, Dict[str, Any]],
 ]:
+
+    from .dagster_dbt_translator import (
+        DagsterDbtTranslator,
+    )
+    dagster_dbt_translator = check.inst_param(
```

Comment:
Might be worthwhile to refactor this as a helper method since this `additional_message` is used everywhere.